setup.py

```diff
@@ -41,7 +41,14 @@
         "requests>=2.22.0",
         "statsd",
         "packaging",
+        "cryptography<3.4,>=3.2",
         "python-digitalocean>=1.16.0",
+        "PyJWT==1.7.1",
```

Comment:
Does this need to be pinned? Can it be `>=1.7.1` instead?

Reply:
azure-cli-core has pinned it to 1.7.1. I tried with `>=1.7.1` and it works. But, there's an error notice in logs while installing the dependencies. I did not want to be the cause of someone's CI/CD pipeline failure :) 

What do you recommend? @achantavy

Reply:
Pin it to 1.7.1 and add a comment on why -- that should be good enough

Reply:
@achantavy I have added a comment and pinned the version. Please review and merge.